--- schema/01.xsd	2017-06-10 04:13:41.113018988 +0200
+++ schema/02.xsd	2017-06-10 04:17:52.713010107 +0200
@@ -99,7 +99,6 @@
   <xsd:complexType name="t_hypervisor">
     <xsd:sequence>
       <xsd:element name="feature" type="t_feature" minOccurs="0" maxOccurs="unbounded" />
-      <xsd:element name="memreq" type="t_memreq" minOccurs="0" maxOccurs="unbounded" />
       <xsd:element name="address_space" type="t_address_space" />
     </xsd:sequence>
     <xsd:attribute name="ncpus" type="number" use="required" />
@@ -120,7 +119,7 @@
     <xsd:attribute name="cpumap" type="list_number" />
     <xsd:attribute name="flags_demand" type="pgprot_list" use="required" />
     <xsd:attribute name="flags_prevent" type="pgprot_list" use="required" />
-    <xsd:attribute name="on" type="xsd:string" />
+    <xsd:attribute name="on" type="xsd:string" use="required" />
   </xsd:complexType>
   <xsd:complexType name="t_address_space">
     <xsd:sequence>
@@ -139,13 +138,12 @@
     <xsd:attribute name="offset" type="number_hex" />
     <xsd:attribute name="subsize" type="number_hex" />
     <xsd:attribute name="cpumap" type="list_number" />
-    <xsd:attribute name="flags" type="pgprot_list" />
+    <xsd:attribute name="flags" type="pgprot_list" use="required" />
     <xsd:attribute name="is_init" type="xsd:string" />
   </xsd:complexType>
 
   <xsd:complexType name="t_guest">
     <xsd:sequence>
-      <xsd:element name="memreq" type="t_memreq" minOccurs="0" maxOccurs="unbounded" />
       <xsd:element name="vdev" type="t_vdev" minOccurs="0" maxOccurs="unbounded" />
       <xsd:element name="address_space" type="t_address_space" />
       <xsd:element name="init" type="t_init" />
